pkg/helm/controller.go

```diff
@@ -402,3 +383,80 @@ func setProxyEnv(job *batch.Job) {
 			envar)
 	}
 }
+
+func contentConfigMap(chart *helmv1.HelmChart) *core.ConfigMap {
+	if chart.Spec.ChartContent == "" {
+		return nil
+	}
+	return &core.ConfigMap{
+		TypeMeta: meta.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: meta.ObjectMeta{
+			Name:      fmt.Sprintf("chart-content-%s", chart.Name),
+			Namespace: chart.Namespace,
+		},
+		Data: map[string]string{
+			fmt.Sprintf("%s.tgz.base64", chart.Name): chart.Spec.ChartContent,
+		},
+	}
+}
+
+func setValuesConfigMap(job *batch.Job, chart *helmv1.HelmChart) *core.ConfigMap {
+	configMap := valuesConfigMap(chart)
+	if configMap == nil {
+		return nil
+	}
+
+	job.Spec.Template.Spec.Volumes = []core.Volume{
+		{
+			Name: "values",
+			VolumeSource: core.VolumeSource{
+				ConfigMap: &core.ConfigMapVolumeSource{
+					LocalObjectReference: core.LocalObjectReference{
+						Name: configMap.Name,
+					},
+				},
+			},
+		},
+	}
+
+	job.Spec.Template.Spec.Containers[0].VolumeMounts = []core.VolumeMount{
```

Comment:
https://github.com/rancher/helm-controller/pull/53/files#r441868717

Reply:
yup thats a bug, going to fix that